lib/module.js

```diff
@@ -123,6 +101,18 @@ module.exports = async function (moduleOptions) {
       }
     }
   })
+
+  /*
+   ** Add /_tailwind UI
+   */
+  const tailwindConfigViewer = require('./serverMiddleware/tailwindConfigViewer')
```

Comment:
`/_tailwind` doesn't need to be expose only if options.exposeConfig is true ?

Reply:
_tailwind exposes the UI for viewing the config values. It doesn't rely on the exposed config file, only the tailwindConfig object that is passed into the `tailwindConfigViewer()` invocation. 

It likely needs to be wrapped in `if (nuxt.options.dev)` or a `if (nuxt.options.dev && options.viewer)` though.

Reply:
Sorry, it's a misunderstanding on my part about `exposeConfig` parameter. 
Is there an interest in exposing the ui in production mode ? 
Maybe it make sense to purpose this option to disable it if needed 👍

Reply:
Should we default viewer to true? 

@Atinux I noticed a `ui` option in the [default options object](https://github.com/nuxt-community/tailwindcss-module/blob/master/lib/module.js#L16). Was that mean't to be the flag for whether the Tailwind colors ui should be enabled? If so, we could use it vs using a `viewer` option.

Reply:
I think we should rename it to `viewer` (`ui` is not even used at the moment).